src/NewTwitchApi/Auth/OauthApi.php

```diff
@@ -121,17 +169,29 @@ public function getAppAccessToken(string $scope = ''): ResponseInterface
     }
 
     /**
+     * Make a request
+     *
+     * @param Request $request
+     * @param array $options
+     * @return ResponseInterface
      * @throws GuzzleException
      */
-    private function makeRequest(Request $request, array $options = []): ResponseInterface
+    protected function makeRequest(Request $request, array $options = []): ResponseInterface
```

Comment:
Why make this (and other methods) protected?

Reply:
I don't see the need to keep it private since we're not extending them anywhere